src/commands/count.js

```diff
@@ -0,0 +1,34 @@
+let count = 0;
+let next = count + 1;
```

Comment:
Should this be inside the function context instead of the global context? I was thinking of maybe using Redis but that may be over-engineering it.

Reply:
Good call. Will move into function context.